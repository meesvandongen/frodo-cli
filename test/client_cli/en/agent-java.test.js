import cp from 'child_process';
import { promisify } from 'util';

const exec = promisify(cp.exec);
const CMD = 'frodo agent java --help';
const { stdout } = await exec(CMD);

test("CLI help interface for 'agent java' should be expected english", async () => {
  expect(stdout).toMatchSnapshot();
});
