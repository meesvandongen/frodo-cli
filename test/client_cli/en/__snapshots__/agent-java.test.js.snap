// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI help interface for 'agent java' should be expected english 1`] = `
"Usage: frodo agent java [options] [command]

Manage java agents.

Options:
  -h, --help      Help

Commands:
  delete          Delete java agents.
  describe        Describe java agents.
  export          Export java agents.
  help [command]  display help for command
  import          Import java agents.
  list            List java agents.
"
`;
